### PR TITLE
feat(api/mime): introduce MIME database

### PR DESCRIPTION
**Pull request by jwerle**

### CODE REVIEW

File: api/internal/monkeypatch.js
Comment (chicoxyzzy) [inline api/internal/monkeypatch.js:1]: how is this related to the mime stuff?

Comment (jwerle) [reply api/internal/monkeypatch.js:1]: oops resolved!

**Review by chicoxyzzy — APPROVED** [review]

**Review by bcomnes — APPROVED** [review]
Feel free to merge if you need this in master before I get to writing tests.
